rcqt.so/src/packageHandlerDeb.cpp

```diff
@@ -18,6 +18,13 @@ PackageHandlerDeb::PackageHandlerDeb(std::string pkgname): PackageHandler{}{
   m_manifest = metaInfo->getFileName();
 }
 
+PackageHandlerDeb::PackageHandlerDeb(): PackageHandler{}{
+  
+  const std::vector<std::string> cmd {std::string("--status"), std::string(""), std::string("--status")};
+
+	metaInfo.reset(new DebPackageInfo(std::string("dpkg"), cmd));
```

Comment:
can depPackageInfo or RPMPackageInfo etc have "dpkg" or "rpm" in their class definition itself? instead of us passing it down? They encapsulate the state specific to that package right? what do you think? We have in ctor, DebPackageInfo which is a Deb specific class and why do we need to still pass dpkg in it as dpkg is specific to Deb class.